islandora_internet_archive_bookreader.module

```diff
@@ -57,6 +65,7 @@ function islandora_internet_archive_bookreader_theme() {
       'variables' => array(
         'object' => NULL,
         'datastream_id' => 'JP2',
+        'backup_datastream_id' => 'JPG',
```

Comment:
Since the old `datastream_id` value as well as this `backup_datastream_id` value no longer get used... this could be breaking functionality in which people have it serving up images from other datastreams?